Packet++/header/NflogLayer.h

```diff
@@ -0,0 +1,264 @@
+#ifndef PACKETPP_NFLOG_LAYER
+#define PACKETPP_NFLOG_LAYER
+
+#include "Layer.h"
+#include "TLVData.h"
+#include "GeneralUtils.h"
```

Comment:
Do we use this include anywhere? If not, we should remove it.
If we only use it in `NflogLayer.cpp` it should move there